[2021-08-28 10:53:39,381] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_C 2021-07-21T00:00:00+00:00 [queued]>
[2021-08-28 10:53:39,384] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: test.training_model_C 2021-07-21T00:00:00+00:00 [queued]>
[2021-08-28 10:53:39,384] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-28 10:53:39,384] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-28 10:53:39,385] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-28 10:53:39,401] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_C> on 2021-07-21T00:00:00+00:00
[2021-08-28 10:53:39,401] {standard_task_runner.py:52} INFO - Started process 1322 to run task
[2021-08-28 10:53:39,403] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'training_model_C', '2021-07-21T00:00:00+00:00', '--job-id', '69', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmp9otagbux', '--error-file', '/tmp/tmppru0iu0y']
[2021-08-28 10:53:39,403] {standard_task_runner.py:77} INFO - Job 69: Subtask training_model_C
[2021-08-28 10:53:39,426] {logging_mixin.py:104} INFO - Running <TaskInstance: test.training_model_C 2021-07-21T00:00:00+00:00 [running]> on host DESKTOP-4B4VMOV.localdomain
[2021-08-28 10:53:39,468] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-07-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-21T00:00:00+00:00
[2021-08-28 10:53:39,468] {python.py:151} INFO - Done. Returned value was: 5
[2021-08-28 10:53:39,488] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=test, task_id=training_model_C, execution_date=20210721T000000, start_date=20210828T085339, end_date=20210828T085339
[2021-08-28 10:53:39,532] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-28 10:53:39,534] {local_task_job.py:149} INFO - Task exited with return code 0
